client/components/UserSettings/ConfigureUserSettingsModal.tsx

```diff
@@ -66,6 +76,21 @@ export const ConfigureUserSettingsModal: React.FC<ModalProps> = ({ show, onClose
         });
     }, [updateSettings, country, interestedInSources, enableEmailNotifications, email]);
 
+    const cancelAlertRef = useRef(null);
+
+    const {
+        isOpen: showAlert,
+        onOpen: openAlert,
+        onClose: closeAlert,
+    } = useDisclosure();
+
+    const onAccountDelete = useCallback(async () => {
+            closeAlert();
+            onClose();
+            logoutUser();
+            deleteUser();
```

Comment:
```suggestion
            await deleteAccount();
            await logoutUser();
```
We should only log out the user if the request was successful. Also, on `logoutUser` the page will be reloaded anyway. No need to close the modals.

Reply:
With this order the account is deleted, then tries to log out and redirect but it returns a 500 error, most likely because the account is already deleted, before the logout. Works when the order is switched to how it used to be though (log out, then delete).